.github/workflows/test.yml

```diff
@@ -20,8 +20,7 @@ jobs:
         with:
           ruby-version: '3.1.4'
 
-      # Rspec fails with bundler v 2.6.x, so we pin it to the latest version that works.
-      - run: gem install bundler -v '2.5.20'
+      - run: gem update bundler
```

Comment:
`h`: Let's please pin to a version. We can update to a more recent one, but I don't like using the latest one by default, because it can break CI when there is a bug in a new version, which is very annoying if you're under pressure trying to release a hotfix or something. We can use the dependabot or something to update the bundler.